AccountService/src/LoginService.java

```diff
@@ -1,21 +1,21 @@
 import java.util.ArrayList;
 
 public class LoginService {
-    CustomerService customerService;
+	CustomerService customerService;
 
-    public LoginService(CustomerService customerService) {
-        this.customerService = customerService;
-    }
+	public LoginService(CustomerService customerService) {
+		this.customerService = customerService;
+	}
 
-    public boolean verifyUser(String username, String password){
-        ArrayList<User> database = customerService.userList;
-        customerService.createData();
+	public boolean verifyUser(String username, String password) {
+		ArrayList<User> database = customerService.userList;
+		customerService.createData();
 
-        for(User user : database) {
-            if(username.equals(user.getUserName()) && password.equals(user.getPassword())){
-                return true;
-            }
-        }
-        return false;
-    }
+		for (User user : database) {
+			if (username.equals(user.getUserName()) && password.equals(user.getPassword())) {
```

Comment:
If an user in our database is created without having Username, we will encounter `nullPointerException` here for `getUserName()`. In this case, we can either add a check to see whether `userName` of the `user` exists, or we can make `userName` and `password` being a final field